xarray/plot/utils.py

```diff
@@ -1148,16 +1148,16 @@ def _adjust_legend_subtitles(legend):
 
 def _infer_meta_data(ds, x, y, hue, hue_style, add_guide, funcname):
     dvars = set(ds.variables.keys())
-    error_msg = " must be one of ({:s})".format(", ".join(dvars))
+    error_msg = f" must be one of ({', '.join(dvars)})"
 
     if x not in dvars:
-        raise ValueError("x" + error_msg)
+        raise ValueError(f"x {error_msg}, got {x}")
```

Comment:
```suggestion
        raise ValueError(f"Expected 'x' {error_msg}. Received {x} instead.")
```

nit: We use this format in a bunch of places. Could be nice to do that here too.